packages/opentelemetry-core/src/platform/node/environment.ts

```diff
@@ -29,6 +31,91 @@ export function getEnv(): Required<ENVIRONMENT> {
   return Object.assign({}, DEFAULT_ENVIRONMENT, processEnv);
 }
 
+/**
+ * Retrieves a number from an environment variable.
+ * - Trims leading and trailing whitespace.
```

Comment:
nit: I suppose could remove this line about trimming. However `Number(...)` effectively does that trimming/ignoring, so fine to keep it.

Reply:
removed in https://github.com/open-telemetry/opentelemetry-js/pull/5443/commits/466cebcdb2163cbaeefd7a54db61ff78f6670a75 🙂